fix(firestore-send-email): added logging for invalid email messages, documentation updated with firestore example (#710)
firebase/extensions: firestore-send-email/README.md

})
```

> Or in Firestore

Reviewer: ```suggestion
Or in Firestore the following fields can be added;
```

| Field       | Type        | Value                |
| ----------- | ----------- | -----------          |

> | to          | String      | someone@example.com  |

Reviewer: ```suggestion
| to          | String      | `someone@example.com`  |
```

> | message     | Map         | {subject: 'Hello from Firebase!', html: 'This is an <code>HTML</code> email body.'} |

Reviewer: ```suggestion
| message     | Map         | `{ subject: 'Hello from Firebase!', html: 'This is an <code>HTML</code> email body.' }` |
```

> | to          | String      | `someone@example.com`  |
> | message     | Map         | `{ subject: 'Hello from Firebase!', html: 'This is an <code>HTML</code> email body.' }` |

Reviewer: ```suggestion
| to          | String      | `"someone@example.com"`  |
| message     | Map         | `{ subject: "Hello from Firebase!", html: "This is an <code>HTML</code> email body." }` |
```

> | Field       | Type        | Value                |
> | ----------- | ----------- | -----------          |
> | to          | String      | `"someone@example.com"`  |
> | message     | Map         | `{ subject: "Hello from Firebase!", html: "This is an <code>HTML</code> email body." }` |

Reviewer: ```suggestion
| Field       | Type        | Value                                                                                   |
| ----------- | ----------- | --------------------------------------------------------------------------------------- |
| to          | String      | `"someone@example.com"`                                                                 |
| message     | Map         | `{ subject: "Hello from Firebase!", html: "This is an <code>HTML</code> email body." }` |
```
